src/types.rs

```diff
@@ -196,6 +196,45 @@ pub struct GetMempoolInfo {
     pub unbroadcastcount: usize,
 }
 
+/// Response from `getrawmempool` with `verbose=true`.
+///
+/// The top-level map key is the txid, and the value contains detailed mempool info per tx.
+pub type GetRawMempoolVerbose = BTreeMap<Txid, MempoolEntry>;
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct MempoolEntry {
+    pub vsize: usize,
+    pub weight: usize,
+    #[serde(default)]
+    pub fee: Option<f64>, // DEPRECATED
+    #[serde(default)]
+    pub modifiedfee: Option<f64>, // DEPRECATED
```

Comment:
NACK: I don't see `modifiedfee` in https://bitcoincore.org/en/doc/29.0.0/rpc/blockchain/getrawmempool/. It should've been `modified` and also use `deserialize_bitcoin`:

```suggestion
    #[serde(default, deserialize_with = "deserialize_bitcoin")]
    pub modified: Option<f64>, // DEPRECATED
```
Also I don't see this as deprecated in https://bitcoincore.org/en/doc/29.0.0/rpc/blockchain/getrawmempool/